.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I don't know how vscode functions, but do we need both `launch.json` files? Also, JSON doesn't have a valid comments format so they should probably be removed from there.

Reply:
As an aside, should `.vcscode` be ignored completely?